packages/components/datetime/src/DateTime.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { CommonProps } from '@contentful/f36-core';
+
+import dayjs, { extend } from 'dayjs';
+import utcPlugin from 'dayjs/plugin/utc';
+extend(utcPlugin);
+
+export interface DateTimeProps extends CommonProps {
```

Comment:
do we want to extend it with Polymorphic<'time'> maybe to obtain all attributes that we would normally have for `<time>` tag?

Reply:
uhmmmm I think so, right? Good idea

Reply:
Ok... I think I don't know how to use the types here

Because we want to allow the user to pass the attributes that the `time` tag has, right?
BUT we do not want to allow them to use `as` prop

Am I going to have to type this without using stuff from `f36-core/Primitive`?

Reply:
it's better to use Polymorphic props here